src/main/java/io/xlate/edi/internal/stream/tokenization/X12Dialect.java

```diff
@@ -94,6 +94,9 @@ boolean initialize(CharacterSet characters) {
 
         version = new String[] { new String(header, 84, 5) };
 
+        if (Character.isAlphabetic(elementRepeater)) {
+            elementRepeater = DFLT_REPETITION_SEPARATOR;
+        }
         if (numericVersion() >= RELEASE_ELEMENT_I65) {
```

Comment:
@hoffrocket, I don't think we can default to the `^` (might clash with other delimiters or with actual data just like the `V` does). That's only the default for EDI data being written. For input, the information in the `ISA` is the only way to get the value. If it's alphabetic, it would be better to disable the `elementRepeater` entirely. What do you think?

```suggestion
        if (numericVersion() >= RELEASE_ELEMENT_I65 && !Character.isAlphabetic(elementRepeater)) {
```

Reply:
Whatever you think is best-- this was a strawman proposal :)

Reply:
If you want to update this PR with the new method in `CharacterSet` and call it from here, I think it should be good to go. I'll get a release out for this as soon as it's merged.